tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "moduleResolution": "node",
     "importHelpers": true,
     "target": "es2015",
-    "module": "esnext",
+    "module": "es2020",
```

Comment:
is this required by ionic or any of the dependencies?

Reply:
Yes this was automatically updated when I update the project to use Angular 10.